bridges/scala-native-0.4/src/main/scala/bloop/scalanative/NativeBridge.scala

```diff
@@ -22,35 +25,40 @@ object NativeBridge {
     if (workdir.isDirectory) Paths.delete(workdir)
     Files.createDirectories(workdir.underlying)
 
-    val nativeLogger = NativeLogger(logger.debug _, logger.info _, logger.warn _, logger.error _)
+    val nativeLogger =
+      build.Logger(logger.trace _, logger.debug _, logger.info _, logger.warn _, logger.error _)
     val config = setUpNativeConfig(project, classpath, config0)
     val nativeMode = config.mode match {
-      case LinkerMode.Debug => Mode.debug
-      case LinkerMode.Release => Mode.release
+      case LinkerMode.Debug => build.Mode.debug
+      case LinkerMode.Release => build.Mode.releaseFast
```

Comment:
What about `build.Mode.releaseFull`? It's a bit aspirational and in most cases `releaseFast` is better, but user probably should still be able to use it.

Reply:
Yes, it is shared between Scala.js and Scala Native.
I chose `releaseFast` because it gives good results without taking forever to link.

Reply:
Then I guess this is ready to be merged @tgodzik ?

Reply:
Sure! I will start working on the release, but most likely it will not happen today - my best estimate is Wednesday.